hir/src/asm/isa.rs

```diff
@@ -1404,6 +1407,18 @@ impl MasmOp {
             Instruction::Debug(DebugOptions::LocalInterval(start, end)) => {
                 Self::DebugFrameRange(unwrap_u16!(start), unwrap_u16!(end))
             }
+            Instruction::Nop => Self::Nop,
+            Instruction::LtImm(imm) => Self::LtImm(unwrap_imm!(imm)),
+            Instruction::LteImm(imm) => Self::LteImm(unwrap_imm!(imm)),
+            Instruction::GtImm(imm) => Self::GtImm(unwrap_imm!(imm)),
+            Instruction::GteImm(imm) => Self::GteImm(unwrap_imm!(imm)),
+            Instruction::U32LtImm(_) => todo!(),
```

Comment:
This should more or less work like the other immediate u32 ops right? Any reason to make these `todo!` here?

Reply:
Done in https://github.com/0xPolygonMiden/compiler/pull/221/commits/13af81c24e7ed03175fd59cd52bc64c1d2c10631